components/layout_2020/quotes.rs

```diff
@@ -0,0 +1,1291 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// Quotes data is obtained from ICU CLDR data file /tmp/cldr-common-46.0.zip.
+
+// TODO(xiaochengh): This file should better be moved to elsewhere and maintained automatically.
+// Or even better, extend the icu create to provide the data directly.
+
+use std::collections::HashMap;
+use std::sync::OnceLock;
+
+use icu_locid::Locale;
+
+#[derive(Clone, Copy, Debug)]
+struct QuotesData(char, char, char, char);
+
+#[derive(Clone, Copy, Debug)]
+pub struct QuotePair {
+    pub opening: char,
+    pub closing: char,
+}
+
+type QuotesMap = HashMap<&'static str, QuotesData>;
```

Comment:
I would just go ahead and remove this typedef as the type itself is not very complicated.

Reply:
Done